packages/ccache/build.sh

```diff
@@ -2,18 +2,23 @@ TERMUX_PKG_HOMEPAGE=https://ccache.samba.org
 TERMUX_PKG_DESCRIPTION="Compiler cache for fast recompilation of C/C++ code"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="4.9.1"
+TERMUX_PKG_VERSION="4.10"
 TERMUX_PKG_SRCURL=https://github.com/ccache/ccache/releases/download/v$TERMUX_PKG_VERSION/ccache-$TERMUX_PKG_VERSION.tar.xz
-TERMUX_PKG_SHA256=4c03bc840699127d16c3f0e6112e3f40ce6a230d5873daa78c60a59c7ef59d25
+TERMUX_PKG_SHA256=83630b5e922b998ab2538823e0cad962c0f956fad1fcf443dd5288269a069660
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="libc++, zlib, zstd"
+TERMUX_PKG_DEPENDS="fmt, libandroid-spawn, libc++, libhiredis, xxhash, zlib, zstd"
```

Comment:
`fmt` should be specified as `BUILD_DEPENDS` instead of `DEPENDS`, because `fmt` package does not ship dynamic link libraries.